src/modules/payment/interface/payment.controller.ts

```
@@ -0,0 +1,124 @@
+import { logger } from '@shared/infrastructure/logging/logger';
+
+export const processPayment = async (req: Request, res: Response) => {
+  try {
+    const {
+      orderId, // Debe venir en el body para asociar el pago a la orden
+      token,
+      issuer_id,
+      payment_method_id,
+      transaction_amount,
+      installments,
+      payer
```

ESLint

[error] Insert `,`

(prettier/prettier)